core/gesture.js

```diff
@@ -166,6 +152,20 @@ Blockly.Gesture = function(e, touchId) {
    * @private
    */
   this.flyout_ = null;
+
+  /**
+   * Boolean for sanity-checking that some code is only called once.
+   * @type {boolean}
+   * @private
+   */
+  this.calledUpdateIsDragging_ = false;
```

Comment:
Do you expect this to stick around in the final version?  I don't necessarily have objections, I'm just curious if you're just trying to catch yourself while writing code or if you expect others to get trapped by this.